docs/config.md

```diff
@@ -197,6 +198,7 @@ Option name               | Description
 `region`                  | Ability to choose different datacenters for run in cloud service. Default value is `null`.
 `headless`                | Ability to run headless browser in cloud service. Default value is `null`.
 `isolation`               | Ability to execute tests in isolated clean-state environment ([incognito browser context](https://chromedevtools.github.io/devtools-protocol/tot/Target/#method-createBrowserContext)). Default value is `false`, but `true` for chrome@93 and higher.
+`passive`                 | Ability to make browser passive in which tests do not run by default. Default value is `false`.
```

Comment:
I would split the first sentence into two.
- Ability to make browser passive
- Tests are not run in passive browsers by default

Currently its hard to read, because it could be read as [passive in which tests do not run by default], like there could be another types of "passive" or something like that